configs/razor_subnets/yolov5_s_spos_shufflenetv2_syncbn_8xb16-300e_coco.py

```diff
@@ -0,0 +1,40 @@
+_base_ = [
+    'mmrazor::_base_/nas_backbones/spos_shufflenet_supernet.py',
+    'mmyolo::yolov5/yolov5_s-v61_syncbn_fast_8xb16-300e_coco.py'
+]
+
+custom_imports = dict(imports=['mmrazor.models'], allow_failed_imports=False)
+checkpoint_file = 'https://download.openmmlab.com/mmrazor/v0.1/nas/spos/spos_shufflenetv2_subnet_8xb128_in1k/spos_shufflenetv2_subnet_8xb128_in1k_flops_0.33M_acc_73.87_20211222-1f0a0b4d.pth'  # noqa
+fix_subnet = 'https://openmmlab-share.oss-cn-hangzhou.aliyuncs.com/mmrazor/v1/spos/yolov5/SPOS_SUBNET.yaml'  # noqa
+widen_factor = 1.0
+channels = [160, 320, 640]
+
+nas_backbone = dict(
+    _delete_=True,
+    type='mmrazor.sub_model',
+    fix_subnet=fix_subnet,
+    cfg=dict(
+        type='mmrazor.SearchableShuffleNetV2',
+        out_indices=(1, 2, 3),
+        arch_setting=_base_.arch_setting,
+        init_cfg=dict(
+            type='Pretrained',
+            checkpoint=checkpoint_file,
+            prefix='architecture.model.backbone.')),
+    extra_prefix='backbone.')
+
+model = dict(
+    backbone=nas_backbone,
+    neck=dict(
+        type='YOLOv5PAFPN',
+        widen_factor=widen_factor,
+        in_channels=channels,
+        out_channels=channels),
+    bbox_head=dict(
+        type='YOLOv5Head',
+        head_module=dict(
+            type='YOLOv5HeadModule',
+            in_channels=channels,
+            widen_factor=widen_factor)))
```

Comment:
```suggestion
_base_.model.backbone = nas_backbone
_base_.model.neck.widen_fator = widen_factor
_base_.model.neck.in_channels = channels
_base_.model.neck.out_channels = channels
_base_.model.bbox_head.head_module.in_channels = channels
_base_.model.bbox_head.head_module.widen_factor = widen_factor
```

Reply:
How about this way? @aptsunny @sunnyxiaohu 

I vote this way because it's more clear to show changes

Reply:
Thanks, will do